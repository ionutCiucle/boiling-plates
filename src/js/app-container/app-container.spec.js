import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AppContainer from './app-container';

enzyme.configure({ adapter: new Adapter() });

describe('<AppContainer/>', function() {
  it('should render correctly', function() {
    const wrapper = shallow(
      <AppContainer/>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});