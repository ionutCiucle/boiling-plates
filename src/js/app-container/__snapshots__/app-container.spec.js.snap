// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppContainer/> should render correctly 1`] = `
<div
  className="app-container"
>
  The plates are boiling
</div>
`;
